### DISCUSSION TITLE
CLI `icon` subcommand not available?

### DISCUSSION THREAD

**Discussion by guyutongxue** [tauri-apps/tauri#3529] [Q&A] [answered]
The `icon` subcommand (documented [here](https://tauri.studio/docs/api/cli#icon)) seems not available in recent versions of Tauri CLI. 
```
> tauri "icon" "public/favicon.png"

error: Found argument 'icon' which wasn't expected, or isn't valid in this context

USAGE:
    pnpm run tauri <SUBCOMMAND>

For more information try --help
```

So how can I generate a series of icons from a given source image now?

**Comment by JonasKruckenberg** [comment, marked as answer]
The `tauri icon` subcommand has been removed at the beginning of the release candidate phase. It depends on several native node modules that were causing installation issues for people. The subcommand is now a standalone CLI: [tauricon](https://github.com/tauri-apps/tauricon).

**Reply by MatiasVerdier** [reply]
Is this supposed to be installed using the GitHub url? The repo doesn't have any instructions.

Thanks in advance

**Reply by FabianLars** [reply]
Yes, you can install it like this until we publish it to npm: `npm install --save-dev github:tauri-apps/tauricon`

**Reply by MatiasVerdier** [reply]
Thanks!

**Reply by Silbad** [reply]
Don't work with Yarn

**Reply by FabianLars** [reply]
@Silbad `yarn add -D github:tauri-apps/tauricon`, then run the command with `yarn tauricon` 

(for npm you also need to add `"tauricon": "tauricon"` to your `scripts` section in package.json)
